docs/tutorials/tutorial-0.rst

```diff
@@ -45,6 +45,8 @@ Check out what the provided ``tutorial_0/app.py`` file contains:
 
 This won't do much as it is, but we can make it useful. 
 
+IF LINUX import __main__ script
```

Comment:
It's not clear what you're trying to say here - there shouldn't be any "linux specific" startup instructions...